.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: 0 0 * * 0
```

Comment:
Why only on Sunday?

Reply:
I put something like this in most of my GHA scripts as a fallback "in case it hasn't run this week, do a run on Sunday when nobody's likely to be looking at it to make sure it's still green" because almost everything I write ends up having to be integration tests (and thus prone to breaking due to external factors that don't necessarily create a commit in the repository).

If GHA had a way to say "run this at a random time Soon if it hasn't run in 14 days" or something like that, I'd happily switch to it. :sweat_smile: